ocaml-lsp-server/docs/ocamllsp/getDocumentation-spec.md

```diff
@@ -0,0 +1,48 @@
+# Documentation Request
+
+## Description
+
+ This custom request allows `odoc` documentation to be gotten without using Hover.
+
+## Client capability
+
+```js
+export interface GetDocClientCapabilities {
+	contentFormat: MarkupKind[];
+}
+```
+- `contentFormat`: Client supports the following content formats if the content property refers to a `literal of type MarkupContent`. The order describes the preferred format of the client.
+
+## Server capability
+
+- property name: `handleDocumentation`
+- property type: `boolean`
+
+
+## Request
+
+```js
+export interface GetDocParams extends TextDocumentPositionParams
+{
+    identifier?: string;
+    contentFormat?:MarkupKind;
+}
+```
+- method : `ocamllsp/getDocumentation`
+- params :
+    - `position`: The position of the cursor.
```

Comment:
Yes, since the interface itself extends `TextDocumentPositionParams` which contains both the Document Uri and the Position

Reply:
We can probably provide a link to the TextDocumentPositionParams and remove the description of `position` (since `textDocument`, the `TextDocumentIdentifier` is not mentionned in the documentation).

Reply:
Not sure that it is right. I think that you should explain that the param should be a TextDocumentPositionParams (with a link to the specification) with additonal ones, enumerated.

Reply:
@xvw I just pushed a commit with some documentation changes. Can you check if it's more structured?